lib/rebaser-server/src/server/rebase.rs

```diff
@@ -102,13 +108,13 @@ pub(crate) async fn perform_rebase(
             info!("pointer updated: {:?}", start.elapsed());
         }
 
-        ReplyRebaseMessage::Success {
-            updates_performed: serde_json::to_value(updates)?,
+        RebaseStatus::Success {
+            updates_performed: serde_json::to_value(updates)?.to_string(),
         }
     } else {
-        ReplyRebaseMessage::ConflictsFound {
-            conflicts_found: serde_json::to_value(conflicts)?,
-            updates_found_and_skipped: serde_json::to_value(updates)?,
+        RebaseStatus::ConflictsFound {
+            conflicts_found: serde_json::to_value(conflicts)?.to_string(),
+            updates_found_and_skipped: serde_json::to_value(updates)?.to_string(),
         }
```

Comment:
For the future: potential use concrete types in the response. We might only need `Conflicts` too.